src/auth-service/models/SurveyResponse.js

```
@@ -0,0 +1,440 @@
+SurveyResponseSchema.index({ surveyId: 1, userId: 1 });
+SurveyResponseSchema.index({ surveyId: 1, status: 1 });
+SurveyResponseSchema.index({ userId: 1, createdAt: -1 });
+
+SurveyResponseSchema.statics = {
+  async register(args, next) {
+    try {
+      let createBody = args;
+
+      // Remove _id if present
+      if (createBody._id) {
+        delete createBody._id;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)